src/tool_getparam.c

```diff
@@ -1198,7 +1198,7 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
         config->ignorecl = toggle;
         break;
       case 'q': /* --ftp-skip-pasv-ip */
-        config->ftp_skip_ip = toggle;
+        config->ftp_pasvp_ip_rule = CURL_FTP_SKIP_PASV_IP_ALWAYS;
```

Comment:
this breaks behavior for users who do `--no-ftp-skip-pasv-ip` to switch off the option